packages/remix-node/package.json

```diff
@@ -15,7 +15,7 @@
     "@remix-run/server-runtime": "1.1.3",
     "@types/busboy": "^0.3.1",
     "@types/node-fetch": "^2.5.12",
-    "@web-std/file": "^3.0.0",
+    "@web-std/file": "^3.0.2",
```

Comment:
Is this update necessary for this PR?
If not, please revert changes to `package.json` & `yarn.lock`

Reply:
It was added by @jacob-ebey when he is verifying the changes for me. I will double check if it is still needed with the latest setup.